language-server/modules/langserver-core/src/test/resources/completion/typedesc_context/config/table_typedesc5a.json

```diff
@@ -4,14 +4,5 @@
     "character": 38
   },
   "source": "typedesc_context/source/table_typedesc5a.bal",
-  "items": [
-    {
-      "label": "recId",
-      "kind": "Field",
-      "detail": "int",
-      "sortText": "G",
-      "insertText": "recId",
-      "insertTextFormat": "Snippet"
-    }
-  ]
+  "items": []
```

Comment:
Not sure how this got affected. This isn't supposed to be changed. This is the source code for this scenario:
```
import ballerina/module1;

public const int TEST_CONST = 12;

public function main() {
    int[] testArr = [];
    table<TableRecord> key(uniqueKey, <cursor>)
    int testValue1 = 12;
}

type Key record {
    string uuid;
};

type TableRecord record {
    readonly int recId;
    string name;
    int age;
    readonly Key uniqueKey;
};
```
And cursor is at `<cursor>` and we should suggest all readonly required fields as a key.

Reply:
Done